plugins/SolidView/SolidView.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2015 Ultimaker B.V.
+# Uranium is released under the terms of the AGPLv3 or higher.
+
+from UM.View.View import View
+from UM.Scene.Iterator.DepthFirstIterator import DepthFirstIterator
+from UM.Resources import Resources
+from UM.Application import Application
+from UM.Math.Color import Color
+from UM.Preferences import Preferences
+from UM.View.Renderer import Renderer
+
+from UM.View.GL.OpenGL import OpenGL
+
+import math
+
+## Standard view for mesh models. 
```

Comment:
How is this different from the simple view? I find solid view as a name a bit confusing (as the models displayed are not solid by definition). Solid models would be the stuff that open-scad generates or some voxel implementation.